src/state/ViewerState.ts

```diff
@@ -58,9 +61,12 @@ export const viewerStateStoreCreator: StateCreator<ViewerState> = (...a) => ({
  * );
  * ```
  */
-export const useViewerStateStore = create<ViewerState>()(subscribeWithSelector(viewerStateStoreCreator));
+export const useViewerStateStore: SubscribableStore<ViewerState> = create<ViewerState>()(
```

Comment:
Used the `SubscribableStore` type here for consistency w/ usage in the `addSubscribers` methods